index.html

```diff
@@ -17,7 +17,7 @@
     frame-src 'self';
     img-src 'self' data: blob: *.hel.fi *.hel.ninja *.blob.core.windows.net;
     font-src 'self' *.hel.fi *.hel.ninja;
-    connect-src 'self' *.hel.fi *.hel.ninja;
+    connect-src 'self' localhost:8081 127.0.0.1:8081 *.hel.fi *.hel.ninja;
```

Comment:
Any port under localhost should be valid.

Reply:
> Any port under localhost should be valid.

We had a discussion and the rationale/need behind this was that @nikomakela has run the backend locally using different ports beside the default 8081.